### PR TITLE
Fix accessibility issues

### PR DESCRIPTION
**Pull request by nickbristow**
Fixes 1,658 accessibility issues, 

I've done some testing, but should be verified by someone more familiar with the app

### CODE REVIEW

**Comment by LindsayYoung** [comment]
This fixes a ton of issues. Thanks again Nick for doing this!

I noticed that it breaks links to anchor tags, I will see if I can get in a quick fix. If not, I can approve this PR and file an issue to fix the anchor links later.

File: src/main/template/operation.handlebars [hunk 1]
```diff
@@ -4,15 +4,17 @@
       <div class='heading'>
         <h3>
           <span class='http_method'>
-          <a href='#!/{{encodedParentId}}/{{nickname}}' class="toggleOperation">+</a>
+          <a href='#{{parentId}}_{{nickname}}' data-href='#!/{{encodedParentId}}/{{nickname}}' class="toggleOperation">+</a>
```
Comment (LindsayYoung) [inline src/main/template/operation.handlebars]: There are 2 hrefs here

Comment (noahmanger) [inline src/main/template/operation.handlebars]: Ah. Well one of those is `data-href`, but i don't see where that's being used. What's this for @nickbristow ?

**Comment by nickbristow** [comment]
@noahmanger @LindsayYoung ok take a look now, I think this is clearer???

**Comment by LindsayYoung** [comment]
Yes, I like this. The section headings don't have links, but the endpoints do, and I think that is more important. 

Making buttons into buttons is brilliant, as far as I am concerned. 

Thanks again!

**Pull request merged**